src/doc/partials/progress-bar-guidance.md.hbs

```diff
@@ -0,0 +1 @@
+* To center the progress bar, the parent control should have `text-align to center`, E.g. `<div style="text-align: center">`.
```

Comment:
@codefactorymx, let's wrap "text-align" and "center" in two separate code spans. Something like: "the parent control should have `text-align` set to `center`, e.g. ..."